### ISSUE TITLE
MobileCoreServices has been renamed. Use CoreServices instead

### ISSUE THREAD

**Issue by colinrf** [BranchMetrics/ios-branch-deep-linking-attribution#1012]
After upgrading to Xcode 11.4 I have new project warnings from Branch.

MobileCoreServices has been renamed. Use CoreServices instead.

**Comment by Ariandr** [comment]
I get the same warning

**Comment by echo-branch** [comment]
We have a backlogged ticket for this warning.  We have to maintain a certain amount of backwards compatibility, so it might be a bit before we can make this update.

**Comment by echo-branch** [comment]
This should be fixed.

**Comment by HermanKayy** [comment]
Is this issue fixed in the latest 0.33 version?

**Comment by echo-branch** [comment]
yes

**Comment by raid5** [comment]
I thought I read in another issue it was slated for the 0.34 release.

**Comment by echo-branch** [comment]
You're probably thinking of another fix that's waiting on staging.  The CoreServices change is in the podspec on master.  

https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/blob/master/Branch.podspec

**Comment by johntmcintosh** [comment]
There was a mention in #999 that this was fixed in the 0.34 release, but I can confirm that I just updated to 0.33.1, and the warning is resolved for us. Thanks!

**Comment by echo-branch** [comment]
Yea, I totally typo'd that comment.  :(
